pkg/util/converter/prom.go

```diff
@@ -8,15 +8,15 @@ import (
 
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
+	sdkjsoniter "github.com/grafana/grafana-plugin-sdk-go/data/utils/jsoniter"
 	jsoniter "github.com/json-iterator/go"
-	"golang.org/x/exp/slices"
 
-	"github.com/grafana/grafana/pkg/util/converter/jsonitere"
+	"golang.org/x/exp/slices"
```

Comment:
nit, but there's an extra newline between imports

Reply:
`make golangci-lint` is not complaining about this one. Since those are imports from different URLs, I think this is ok.